### PR TITLE
Added JuliaHealth GSoC Draft Proposal

### PR DESCRIPTION
**Pull request by TheCedarPrince**
Feel free to merge this if urgent. Otherwise, will do a final revision in the next day or two. Thanks for the pings @logankilpatrick !

CC @DilumAluthge as well!

### CODE REVIEW

**Review by DilumAluthge — APPROVED** [review]

File: jsoc/gsoc/juliahealth.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+- **Difficulty**: Hard
+- **Duration**: 175 hours
+- **Suggested Skills and Background**: 
+    - Strong background in machine learning
+    - Familiarity with the OMOP Common Data Model
+    - Experience with Julia
+  Patient level prediction (PLP) is an important area of research in healthcare that involves using patient data to predict outcomes such as disease progression, response to treatment, and hospital readmissions
+JuliaHealth is interested in developing tooling for PLP that utilizes data in the OMOP CDM format
+For this project, we are looking for a student who is interested in developing PLP tooling within the JuliaHealth organization
+This could involve creating new packages that implement state-of-the-art PLP algorithms, improving the performance and scalability of existing tools, or developing new visualization tools to help researchers better understand their results.
```
Comment (logankilpatrick) [inline jsoc/gsoc/juliahealth.md:35]: Can we get more specific here? and with the other projects? It should be clear what the deliverable is for someone interested in this.

**Comment by logankilpatrick** [comment]
@TheCedarPrince can you also add the JuliaHealth page to the list of projects, it's a separate file that lists them all.

**Comment by TheCedarPrince** [comment]
Absolutely @logankilpatrick ! i will get to this all this evening after work. :smile:

File: jsoc/gsoc/juliahealth.md (hunk 1 above)
Comment (TheCedarPrince) [reply jsoc/gsoc/juliahealth.md:35]: This should now be taken care of!

**Comment by logankilpatrick** [comment]
Thank you both!

**Pull request merged**